src/dotnet-test-xunit/Program.cs

```diff
@@ -82,6 +81,15 @@ public int Run(string[] args)
                 if (!commandLine.NoLogo)
                     PrintHeader();
 
+                if (commandLine.Port.HasValue)
```

Comment:
I did it like this to leave the "old" way of doing it working still. In my mind if simplified the compatibility matrix.